test/test_meta.py

```diff
@@ -1023,6 +1027,66 @@ def test_fill_alias_relationship(self):
         r2 = torch.ops.aten.fill(inps, 1.0)
         self.assertNotEqual(id(inps), id(r2))
 
+    def get_stride_variants(self, t):
```

Comment:
cool! Should we extend all meta tests with inputs generated like this?

Reply:
Yes, we can generate strided inputs from existing opinfo test cases.